torch/CMakeLists.txt

```diff
@@ -108,10 +108,17 @@ set(TORCH_PYTHON_SRCS
     ${TORCH_SRC_DIR}/csrc/utils/tensor_new.cpp
     ${TORCH_SRC_DIR}/csrc/utils/tensor_numpy.cpp
     ${TORCH_SRC_DIR}/csrc/utils/tensor_types.cpp
-    ${TORCH_ROOT}/test/cpp/jit/torch_python_test.cpp
-    ${JIT_TEST_SRCS}
     )
 
+# NB: This has to match the condition under which the JIT test directory
+#     is included (at the time of writing that's in caffe2/CMakeLists.txt).
+if (BUILD_TEST AND NOT MSVC AND NOT USE_ROCM)
```

Comment:
Why is this disabled for MSVC?

Reply:
@apaszke would know best, but judging from the comment above, I think he was channeling the fact that JIT tests are already disabled on Windows:

```

  if (BUILD_TEST AND NOT MSVC AND NOT USE_ROCM)
    add_subdirectory(${TORCH_ROOT}/test/cpp/jit ${CMAKE_BINARY_DIR}/test_jit)
    if (USE_DISTRIBUTED)
      add_subdirectory(${TORCH_ROOT}/test/cpp/rpc ${CMAKE_BINARY_DIR}/test_cpp_rpc)
    endif()
  endif()
```

But why is this disabled on Windows? The condition was added in #8792; we can't easily ask Anders why he added it, but probably at the time it was failing on Windows and so he disabled it instead of fixing it.